app/src/main/java/ch/epfl/sweng/swenggolf/offer/list/ListOfferAdapter.java

```diff
@@ -75,13 +78,26 @@ public int getItemCount() {
     public boolean add(@NonNull List<Offer> offers) {
         if (!offers.isEmpty()) {
             offerList.addAll(offers);
+            Collections.sort(offerList,creationDateComparator);
             filteredOfferList.clear();
             fillFilteredList();
             notifyDataSetChanged();
         }
         return filteredOfferList.isEmpty();
     }
 
+    private static final Comparator<Offer> creationDateComparator = new Comparator<Offer>() {
```

Comment:
You could add a comment saying in which order the Offer are sorted (more recent at the beginning).